includes/admin.form.inc

```diff
@@ -127,6 +127,16 @@ function islandora_repository_admin(array $form, array &$form_state) {
             ),
           ),
         ),
+        'islandora_orphaned_objects_backend' => array(
+          '#type' => 'radios',
+          '#title' => t('Orphaned Objects query'),
+          '#description' => t('How the Orphaned Islandora Objects list is generated.'),
+          '#default_value' => variable_get('islandora_orphaned_objects_backend', 'Solr'),
```

Comment:
This line is setting Solr as the default value for the `islandora_orphaned_objects_backend` variable, which is bad. Any time you add something new, you should always leave the default value to be the old way so that you don't mess anything up. Creating a brand new variable with a default value of the NEW way of doing things means that the way things work for people NOW will change without them understanding why.

All this to say this line needs to change to `variable_get('islandora_orphaned_objects_backend', 'SPARQL')`.